api/src/main/resources/workbench.yaml

```diff
@@ -684,6 +685,29 @@ definitions:
         type: integer
         format: int64
         description: Milliseconds since the UNIX epoch.
+      UserRoles:
```

Comment:
Yes, it should be `userRoles` (I made that fix, but haven't pushed it yet).
I don't see a reason why it shouldn't just be [], that is an artifact from when I was returning these as a response.

Reply:
I'm not sure if this is the same question David asked, but: Why have a `UserRoleList`, instead of just making the `Workspace.userRoles` property `type: "array"` (like the `UserRoleList.items` property itself)?

Reply:
I need to pass up a UserRoleList for actually sharing, I believe? (Unless I just pass up the workspace object itself, which feels a bit misleading.)

Reply:
You could pass up a Workspace with PATCH semantics (just specify the userRoles field value). But I'm fine with saying we explicitly represent a UserRoleList as a thing, too.